mappings/net/minecraft/client/world/WorldClient.mapping

```diff
@@ -7,12 +7,16 @@ CLASS bqk net/minecraft/client/world/WorldClient
 	METHOD a playSound (DDDLpb;Lpd;FFZ)V
 	METHOD a getEntityById (I)Lub;
 	METHOD a isChunkLoaded (IIZ)Z
+		ARG 0 chunkX
+		ARG 1 chunkZ
+		ARG 2 consultRemoteProvider
```

Comment:
Yeah, me and @asiekierka were discussing this one - it's a difficult parameter, because it's essentially an override - if the variable is true, the ChunkProvider isn't consulted on the clientside, with no bearing on the serverside. In hindsight, it probably needs to be renamed anyway, as it's sorta the opposite of what I named it...